htdocs/core/actions_sendmails.inc.php

```diff
@@ -382,7 +382,7 @@
 					// Initialisation of datas of object to call trigger
 					if (is_object($object)) {
 						if (empty($actiontypecode)) {
-							$actiontypecode = 'AC_OTH_AUTO'; // Event insert into agenda automatically
+							$actiontypecode = 'AC_EMAIL'; // Event insert into agenda automatically
```

Comment:
There is 2 types of events:
Event enter manually by the user into the agenda.
Events created automatically by the system for tracking purposes.

An email send from a screen is the second one. So it must not appears into the agenda of the user among its rendez vous but into the agenda of tracking events. So $actiontypecode (the type auto / non auto) must be 'AC_OTH_AUTO'.
There is a column "code" into the table llx_actioncomm that details what king of action has been recorded. Emails are codes like 'AC_..._SENTBYMAIL'